how-to/business-logic/how-to-determine-requested-item-id.md

```diff
@@ -0,0 +1,55 @@
+---
+title: How to Determine the Id of the Requested Item
+description: An example to determine the Id of the requested item

```

Comment:
Consider adding some information about the services and APIs employed because this is too generic right now.